lua/cfc_ulx_commands/curse/effects/crouch.lua

```diff
@@ -0,0 +1,26 @@
+local EFFECT_NAME = "Crouch"
```

Comment:
Might be a bit boring, something like `cmd:CommandNumber() % 4 == 0` would probably spice it up